### PR TITLE
Add GO and Objective-C examples

### PR DESCRIPTION
**Pull request by c410-f3r**
Should wait for the API changes of #114 before merging

### CODE REVIEW

File: examples/ffi/go/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# GO bindingds
```
Comment (jleni) [inline examples/ffi/go/README.md:1]:
```suggestion
# GO bindings
```

**Comment by jleni** [comment]
Nice work!

**Comment by c410-f3r** [comment]
Some updated dependency broke the build. Probably `forest`

**Comment by jleni** [comment]
> Some updated dependency broke the build. Probably `forest`

Yes changes upstream in forest. Fixing that right now

**Comment by jleni** [comment]
This should go back to normal once https://github.com/Zondax/filecoin-rs/pull/117 is merged

**Pull request merged**